src/System.Management.Automation/utils/PathUtils.cs

```diff
@@ -188,6 +188,12 @@ internal static void ReportFileOpenFailure(Cmdlet cmdlet, string filePath, Excep
             cmdlet.ThrowTerminatingError(errorRecord);
         }
 
+        internal static StreamReader OpenStreamReader(PSCmdlet command, string filePath, Encoding encoding, bool isLiteralPath)
+        {
+            FileStream fileStream = OpenFileStream(filePath, command, isLiteralPath);
+            return new StreamReader(fileStream, encoding);
+        }
+
         internal static StreamReader OpenStreamReader(PSCmdlet command, string filePath, string encoding, bool isLiteralPath)
```

Comment:
Is this method still used anywhere? It looks to me not used anymore. If that's the case, then it's better to remove it.

Reply:
removed